apps/public-reference/lib/ApplicationConductor.ts

```diff
@@ -41,8 +43,12 @@ export default class ApplicationConductor {
     return 5
   }
 
-  advanceToNextStep() {
-    this.application.completedStep += 1
+  completeStep(step) {
+    this.application.completedStep = Math.max(step, this.application.completedStep)
+  }
+
+  canJumpForwardToReview() {
+    return this.application.completedStep == 4
```

Comment:
Can that become `return this.application.completedStep == totalNumberOfSteps() -1`?